### PR TITLE
Fixes #619: Update Page permissions

### LINKED ISSUE #619: Update Page permissions
Motivation: I noticed that content admins/editors could not edit menu options when editing a page.

Content editors should be able to access:
- Revision info
- Menu settings
- URL alias
- URL redirects
- Promotion options

Content admins should be able to access:
- XML sitemap
- Authoring information

### PR DESCRIPTION
**Pull request by danahertzberg**
## Description
Allowed menu, xml and redirect settings for content admins/editors

## Related Issue
Closes #619 

## How to test

Log in as **content editor**

1. Edit or add a page
2. View newly added menu settings on right sidebar

Log in as **content admin**
1. Edit or add a page
2. View newly added menu, xml sitemap, authoring, promotion settings on right sidebar

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by smayden** [comment]
One question based on what is outlined in #619:
-I didn't see content editors having promotion options in my testing (but was listed in the ticket)

And a few questions overall about what permissions should be for content editors vs. content admins
-Should content editors have access to edit menus? I don't see that permission for them on the few QS1 sites I have access to (mainly quickstart.arizona.edu). I've seen content editors add things to main navigation without even realizing it on other Drupal sites... 
-Similarly, should content editors have access to make redirects? Seems to make more sense for content admins and I don't see that as a current option on some QS 1/UA Sites where I'm a content editor

**Comment by danahertzberg** [comment]
@smayden Thanks for the feedback!
- Looks like an error on my part that I put promotion options for content admins instead of content editors. I will update.
- We allow all our content editors to edit menus. I am open to restricting that access, though.
- Same fore redirects, we allow it on QS1. But I'm open to having more differences between content admins/editors.

In terms of what we should/shouldn't do; I'm all for having higher restrictions now. And if loosening them in the future is needed, that's an easier fix than taking away permissions.

**Comment by danahertzberg** [comment]
I updated the permissions as we discussed. Looks like the promotion permissions are part of a larger permission setting that we probably don't want content editors to have so I left it with only content admins.

Permission setting with promotion:
```
Administer content
Warning: Give to trusted roles only; this permission has security implications. Promote, change ownership, edit revisions, and perform other tasks across all content types.
```

**Review by smayden — APPROVED** [review]
Tested as both content editor and content admin and all working as expected

**Review by joeparsons — APPROVED** [review]

**Review by tadean — DISMISSED** [review]

File: config/install/user.role.az_content_admin.yml [hunk 1]
```diff
@@ -10,9 +10,13 @@ permissions:
   - 'access taxonomy overview'
   - 'administer book outlines'
   - 'administer easy breadcrumb'
+  - 'administer menu'
   - 'administer nodes'
   - 'administer quickstart configuration'
+  - 'administer redirect settings'
```
Comment (danahertzberg) [inline config/install/user.role.az_content_admin.yml:16]:
```suggestion
```

**Review by joeparsons — APPROVED** [review]

**Pull request merged**